man/ocamlrun.m

```diff
@@ -69,22 +69,45 @@
 .B b
 flag in the OCAMLRUNPARAM environment variable (see below).
 .TP
+.B \-H
+Allocates heap chunks by mmapping huge pages. Huge pages are locked into
+memory, and are not swapped.
+.TP
 .BI \-I \ dir
 Search the directory
 .I dir
 for dynamically-loaded libraries, in addition to the standard search path.
 .TP
+.B \-m
+Print the magic number of the bytecode executable given as argument and
+exit.
+.TP
+.B \-M
+Print the magic number expected for bytecode executable by this version
+of the runtime and exit.
```

Comment:
bytecode executable**s** - this text should also be updated in `manual/manual/cmds/runtime.etex`